src/libs/actions/IOU.ts

```diff
@@ -6526,7 +6526,10 @@ function getReportFromHoldRequestsOnyxData(
         {
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT}${optimisticExpenseReport.reportID}`,
-            value: optimisticExpenseReport,
+            value: {
+                ...optimisticExpenseReport,
+                unheldTotal: 0,
```

Comment:
Adding this caused https://github.com/Expensify/App/issues/50932, although this only happened in combination with https://github.com/Expensify/App/pull/50817. Essentially we needed to handle the case where the approval is for a manual expense.